cli/things.go

```diff
@@ -189,7 +189,7 @@ var cmdThings = []cobra.Command{
 			pm := mfxsdk.PageMetadata{
 				Offset:       uint64(Offset),
 				Limit:        uint64(Limit),
-				Disconnected: true,
+				Disconnected: false,
```

Comment:
Same here, why are you inverting the value if it's to list `non-connected` ?

Reply:
Here `disconnected` should be `false` meaning `connected` it `true`